### PR TITLE
Vdb 1673 remove unused sub commands

### PR DESCRIPTION
**Pull request by yaoandrew**
This removes `composeAndExecute` as well as the `contractWatcher`. By removing `contractWatcher` we can also get rid of some integrationtests and the `contract_watcher` package. There are potentially more things to delete if we modify the `Exporter` interface which spans both this repo and the `transformers` repo.

`ContractTransformer` interface and initializer are still there since its returned by the `Exporter`
`ContractConfig` is also still in the interface

### CODE REVIEW

File: pkg/fakes/parser.go [hunk 1]
```diff
@@ -18,13 +18,11 @@ package fakes
 
 import (
 	"github.com/ethereum/go-ethereum/accounts/abi"
-	"github.com/makerdao/vulcanizedb/pkg/contract_watcher/types"
 )
 
 type MockParser struct {
 	AbiToReturn string
 	EventName   string
```
Comment (rmulhol) [inline pkg/fakes/parser.go]: Can't delete this whole file?

**Review by rmulhol — COMMENTED** [review]
Looking good - I'm wondering if we might also want to make some updates to the docs?

File: pkg/fakes/parser.go (hunk 1 above)
Comment (yaoandrew) [reply pkg/fakes/parser.go]: 👍  Ah yes... Miss this one.

**Comment by yaoandrew** [comment]
> Looking good - I'm wondering if we might also want to make some updates to the docs?

Yes. I think I'll come back around to this before we merge. There are a lot of refs to the `contract_watcher` that can go.

**Comment by rmulhol** [comment]
> > Looking good - I'm wondering if we might also want to make some updates to the docs?
> 
> Yes. I think I'll come back around to this before we merge. There are a lot of refs to the `contract_watcher` that can go.

Cool - lmk whenever that's taken care of and I'm happy to give this a review

**Review by rmulhol — APPROVED** [review]

**Pull request merged**